ide/static/js/netLayout_vertical.js

```diff
@@ -33,6 +33,9 @@ function allocatePosition(layerId, preferredPosition){
   if (!map.hasOwnProperty(preferredPosition[1])) {
     map[preferredPosition[1]] = [];
   }
+    
+  map[0]=[];
```

Comment:
map[0] was not define earlier(because preferredPosition[1] can never be 0) and because of that it was showing error in later code(map[preferredPosition[1]-1]).I dont really know why this error was not found earlier.May be because this model is unique and has 2 input sources.

Reply:
Good catch, thanks!